uint8 ID_ALL=0 # Reset ALL configuration to factory default
uint8 ID_NTP=1 # Reset NTP configuration to factory default

uint8 id